pixiedust/utils/sampleData.py

```diff
@@ -227,7 +228,8 @@ def download(self, dataLoader):
             url = self.dataDef["url"]
             req = Request(url, None, self.headers)
             print("Downloading '{0}' from {1}".format(displayName, url))
-            with tempfile.NamedTemporaryFile(delete=False) as f:
+            tdir = '/home/spark/shared' if Environment.hasSpark and not self.forcePandas and os.path.exists('/home/spark/shared') else '/tmp'
```

Comment:
What about platforms where /tmp doesn't exist, like windows maybe?
Perhaps we should use the temp module to get the temp directory in a platform agnostic manner e.g.
tdir = '/home/spark/shared' if Environment.hasSpark and not self.forcePandas and os.path.exists('/home/spark/shared') else tempfile.gettempdir()

Reply:
my bad, good catch! the changes have been made, thanks.